chart/templates/deployment.yaml

```diff
@@ -156,6 +156,20 @@ spec:
                   name: {{ include "obot.config.secretName" . }}
                   key: workspaceProviderS3Bucket
             {{- end }}
+            {{- if .Values.config.workspaceProviderAzureContainer }}
```

Comment:
Can you rebase the chart changes for this off of https://github.com/obot-platform/obot/pull/1965

Reply:
Sorry for the delay on this. I just rebased it